modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1104 @@
+
+As part of that fix, a change was made to the `fullscreen` plugin.
+
+As of {productname} 6.1, toggling fullscreen mode with the `fullscreen` plugin also fires the `ResizeEditor` event.
+
+
+=== Getting the editor’s text content now returns newlines instead of an empty string if more than one empty paragraph exists
+
+Previously, if a {productname} editor instance contained nothing but blank lines (ie, more than one Return but no other material), the plain text version of the editor’s contents was an empty string.
+
+In {productname} 6.1, the plain text version of the same material is two newline characters — `+\n\n+` — for each entered Return.
+
+NOTE: In the HTML version of such a document (viewable by choosing *View > Source Code*), each Enter or Return was represented by a paragraph tag containing a non-breaking space entity: `+<p>&nbsp;</p>+`. This representation, which accurately presents the empty paragraphs as white-space is unchanged in {productname} 6.1.
+
+
+=== The `end_container_on_empty_block` option can now take a string of blocks, allowing the exiting of a blockquote element by pressing Enter or Return twice
+
+Previously, when the insertion point was within a `blockquote` element, and the `blockquote` element was the only block element in the {productname} document, there was no way of exiting the element without deleting all the text also within the element.
+
+As of this update, the `end_container_on_empty_block` has been changed such that pressing the *Enter* or *Return* key twice will exit the insertion point from a `blockquote` element. This is the same action that exits other block elements, such as lists.
+
+
+=== The default value for `end_container_on_empty_block` option has been changed to `blockquote`
+
+Previously, the `end_container_on_empty_block` took a boolean value and, by default, that value was `false`.
+
+As part of the changes made to enable the _pressing Enter or Return twice exits an in-document `blockquote` element_ change noted above, this option’s default value has been changed.
+
+As of {productname} 6.1, `end_container_on_empty_block` takes the value `blockquote` by default.
+
+NOTE: The boolean values, `true` and `false` remain valid values to assign to this option.
+
+
+=== Custom elements are now treated as non-empty elements by the schema
+
+Previously {productname} treated custom elements as empty elements. The {productname} schema was such that the `isEmpty` check returned `true` for all custom elements.
+
+With this update the {productname} schema has been updated; the `isEmpty` check now returns `false` for custom elements. 
+
+
+=== The autocompleter's menu HTML element is now positioned instead of the wrapper
+
+The `autocompleter` menu element presents a menu of formatting options. If the current viewport is too small to present the entire menu, access to the ‘overflow’ is ensured by presenting a scrollbar in the menu.
```

Comment:
This still isn't quite correct as it can be any configured menu item.
```suggestion
The `autocompleter` menu element presents a menu of related options. If the current viewport is too small to present the entire menu, access to the ‘overflow’ is ensured by presenting a scrollbar in the menu.
```